cli/ballerina-cli-utils/pom.xml

```diff
@@ -139,6 +139,7 @@
                             </systemProperties>
                             <arguments>
                                 <argument>${ballerina.source.directory}/packaging_pull</argument>
+                                <argument>packaging_pull.bal</argument>
```

Comment:
Why we have added the bal file as argument now? Previously did it work without that bal file as argument?

Reply:
Previously we just gave the package name and did a `ballerina build`, then the balx was created inside the same directory as the source. Now  since we are creating the balx inside the current directory where the user is in, we should pass the target file name as well. So in this case we need to pass the bal file as an argument since we are using `ballerina build <source-file> -o <target-path>`